libyui-qt-pkg/src/YQPkgPatternList.cc

```diff
@@ -361,6 +376,19 @@ YQPkgPatternListItem::init()
 
 	setIcon( _patternList->iconCol(), YQUI::ui()->loadIcon( iconName ) );
 
+        if ( _patternList->showInvisiblePatterns() && ! _zyppPattern->userVisible() )
+        {
+            // The YQPkgObjListItem base class takes care of setting the name
+            // and summary columns, but here we want to add something to it.
+            // Notice that patterns use the summary column.
+
+            QString name = text( _patternList->summaryCol() );
+            name += " [Invisible]";
```

Comment:
Although this is used only in a special debugging case maybe we should still mark it for translation... :thinking:

Reply:
I am not so sure about that... it's a very special environment.

Reply:
Against my better judgement...
But I had to force-push to keep the history clean and understandable.